pkg/system/rm.go

```diff
@@ -29,7 +30,9 @@ func EnsureRemoveAll(dir string) error {
 	maxRetry := 100
 
 	// Attempt to unmount anything beneath this dir first
-	mount.RecursiveUnmount(dir)
+	if err := mount.RecursiveUnmount(dir); err != nil {
+		logrus.Debugf("RecusiveUnmount on %s failed: %v", dir, err)
```

Comment:
Since commit 8b67934cbcc1df the unmount error is verbose enough per se, producing something like

`umount <dir>, flags 0x01: device busy`

so you don't have to print `dir` anymore. IOW

```suggestion
		logrus.Debugf("EnsureRemoveAll warning: %v", err)
```